playground/BrowserTelemetry/BrowserTelemetry.Web/BrowserTelemetry.Web.csproj

```diff
@@ -4,13 +4,15 @@
     <TargetFramework>net8.0</TargetFramework>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
+    <!-- Issue: https://github.com/dotnet/dnceng/issues/3091 -->
+    <NpmAvailable Condition="'$(ContinuousIntegrationBuild)' != 'true' and '$(OS)' != 'Windows_NT'">true</NpmAvailable>
```

Comment:
Should this be

```suggestion
    <NpmAvailable Condition="'$(ContinuousIntegrationBuild)' != 'true' or '$(OS)' != 'Windows_NT'">true</NpmAvailable>
```

Reply:
I should invert this to set when `Npm` is *not* available, because that's all we are trying to say.